tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36-tests
+envlist = py36-tests,py37-tests,py38-tests,py39-tests
```

I think this should work:
```suggestion
envlist = {py36,py37,py38,py39}-tests
```

---

Yeah it does. This is how we are generating the versions from the script, although this is happening in the Makefile now